src/xrCore/xrCore.vcxproj

```diff
@@ -18,6 +18,7 @@
     <ProjectGuid>{A0F7D1FB-59A7-4717-A7E4-96F37E91998E}</ProjectGuid>
     <RootNamespace>xrCore</RootNamespace>
     <Keyword>Win32Proj</Keyword>
+    <WindowsTargetPlatformVersion>8.1</WindowsTargetPlatformVersion>
```

Comment:
Windows target version is unset for all the project files. This is not critical, but it's recommended to not change this.